packages/web/src/components/collections-table/CollectionsTable.tsx

```diff
@@ -0,0 +1,263 @@
+import { MouseEvent, useCallback, useMemo, useRef } from 'react'
+
+import {
+  CollectionMetadata,
+  UserCollectionMetadata
+} from '@audius/common/models'
+import { formatCount } from '@audius/common/utils'
+import { Flex } from '@audius/harmony'
+import cn from 'classnames'
+import moment from 'moment'
+import { Cell, Row } from 'react-table'
+
+import { TextLink } from 'components/link'
+import { Table, alphaSorter, dateSorter, numericSorter } from 'components/table'
+
+import styles from './CollectionsTable.module.css'
+import { CollectionsTableOverflowMenuButton } from './CollectionsTableOverflowMenuButton'
+
+type RowInfo = UserCollectionMetadata & {
+  name: string
+  artist: string
+  date: string
+  dateSaved: string
+  dateAdded: string
+}
+
+type CollectionCell = Cell<RowInfo>
+
+type CollectionRow = Row<RowInfo>
+
+export type CollectionsTableColumn =
+  | 'name'
+  | 'overflowMenu'
+  | 'releaseDate'
+  | 'reposts'
+  | 'saves'
+  | 'spacer'
+
+type CollectionsTableProps = {
+  columns?: CollectionsTableColumn[]
+  data: any[]
+  isVirtualized?: boolean
+  loading?: boolean
+  defaultSorter?: (a: any, b: any) => number
+  fetchMore?: (offset: number, limit: number) => void
+  fetchBatchSize?: number
+  onClickRow?: (collection: CollectionMetadata, index: number) => void
+  scrollRef?: React.MutableRefObject<HTMLDivElement | undefined>
+  showMoreLimit?: number
+  totalRowCount?: number
+}
+
+const defaultColumns: CollectionsTableColumn[] = [
+  'name',
+  'overflowMenu',
+  'releaseDate',
+  'reposts',
+  'saves',
+  'spacer'
+]
+
+/**
+ * Note: this table is not generalized the way TracksTable is. It is
+ * specifically tailored to the Artist Dashboard use case.
+ */
+export const CollectionsTable = ({
+  columns = defaultColumns,
+  data,
+  defaultSorter,
+  fetchBatchSize,
+  fetchMore,
+  isVirtualized = false,
+  loading = false,
+  onClickRow,
+  scrollRef,
+  showMoreLimit,
+  totalRowCount
+}: CollectionsTableProps) => {
+  // Cell Render Functions
+  const renderNameCell = useCallback((cellInfo: CollectionCell) => {
+    const collection = cellInfo.row.original
+    const deleted = collection.is_delete || !!collection.user?.is_deactivated
+
+    return (
+      <Flex
+        gap='xs'
+        css={{
+          overflow: 'hidden',
+          position: 'relative',
+          display: 'inline-flex',
+          'max-width': '100%'
+        }}
+      >
+        <TextLink
+          tag={deleted ? 'span' : undefined}
```

Comment:
maybe just conditionally render the span vs link instead of swapping the tag?

Reply:
tbh this is copied from [here](https://github.com/AudiusProject/audius-protocol/blob/8feb22bf3517625f0c8e83bd4786829e119a313f/packages/web/src/components/collection/desktop/OverflowMenuButton.tsx). would prefer to leave as-is bc then we're conditionally rendering two diff components wrapping the same text, which is functionally the same as what we're doing here.